codes/java/.gitignore

```diff
@@ -0,0 +1 @@
+/.metadata/
```

Comment:
`.metadata` is enough for your purpose, or `codes/jave/.metadata` if you want to make it accurate